packages/big-design/src/components/TableNext/Row/Row.tsx

```diff
@@ -154,7 +161,11 @@ const InternalRow = <T extends TableItem>({
               verticalAlign={verticalAlign}
               width={isDragging ? cellWidth : width}
             >
-              <Flex alignItems="center" flexDirection="row">
+              <Flex
+                alignItems="center"
+                flexDirection="row"
+                justifyContent={align && ALIGN_MAP[align]}
+              >
```

Comment:
Does this mean that the `align` property in the `DataCell` is not used. Should we remove it altogether?

@MariaJose Do you remember what is the reasoning behind adding the `Flex` container?

Reply:
Let me check it. Also I will check thead cells

Reply:
@jorgemoya  It was to fix alignment and space issues when the children rows use to have the checkbox component as well. But since we remove the checkbox for the children for now, we might not need the Flex component.

Reference: https://github.com/bigcommerce/big-design/pull/934/files

Reply:
We will add the feature back, so let's persist the change if it resolved a styling issue.

Reply:
cool with that ^ 👍🏼

Reply:
@jorgemoya thead cells working right.

> Does this mean that the align property in the DataCell is not used. Should we remove it altogether?

We need it for typography because the `align` property adds `text-align`.